### ISSUE TITLE
Exception from Tracker recompute

### ISSUE THREAD

**Issue by almottier** [kromitgmbh/titra#40]
I have these error when I edit a project
```
ERROR: Exception from Tracker recompute function: @ Tue Dec 22 2020 19:12:09 GMT+0000 (Coordinated Universal Time)] {
titra         |   "browserInfo": {
titra         |     "browser": "Mozilla/5.0...
```

```
titra         | [ERROR: TypeError: Cannot read property 'match' of undefined @ Tue Dec 22 2020 19:12:09 GMT+0000 (Coordinated Universal Time)] {
```

**Comment by faburem** [comment]
Thanks for reporting this issue, unfortunately I don't seem to be able to reproduce it. Can you please provide a little bit more information on the steps needed to reproduce the problem?
Other interesting information includes:
- the version of titra running on the server

**Comment by almottier** [comment]
I use Docker on Ubuntu 20 machine, the image is `kromit/titra`. 
I have this error when editing a Project.

**Comment by faburem** [comment]
Can you please post a screenshot of the titra About page so know which version of titra you are running?
![image](https://user-images.githubusercontent.com/11456790/103013982-56b62a80-453e-11eb-82e0-e172d0e2a254.png)

Thanks and merry Christmas,
Fabian

**Comment by almottier** [comment]
Here it is. v0.32

![Screenshot_20201223-193934~2.png](https://user-images.githubusercontent.com/5426740/103037029-53856380-456b-11eb-9efe-0d8292169671.png)

Merry Christmas 🌲😊

**Comment by faburem** [comment]
Can you please try if you still reproduce the issue in titra version 0.34.0?
Thanks for your feedback!

**Comment by almottier** [comment]
Yes, same problem with 0.34.0

```text
[ERROR: Exception from Tracker recompute function: @ Sun Dec 27 2020 21:33:20 GMT+0000 (Coordinated Universal Time)] {
  "browserInfo": {
    "browser": "Mozilla/5.0 (Macintosh; Intel Mac OS X) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36",
    "userId": "mg7L9xkiyMYFEuHTD",
    "url": "https://xxx/edit/project/YuFEvRZmX3h85bxs6",
    "resolution": "1680x1050"
  },
  "stack": []
}
[ERROR: TypeError: Cannot read property 'match' of undefined @ Sun Dec 27 2020 21:33:20 GMT+0000 (Coordinated Universal Time)] {
  "browserInfo": {
    "browser": "Mozilla/5.0 (Macintosh; Intel Mac OS X) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36",
    "userId": "mg7L9xkiyMYFEuHTD",
    "url": "https://xxx/edit/project/YuFEvRZmX3h85bxs6",
    "resolution": "1680x1050"
  },
  "stack": [
    "at A (/node_modules/@simonwep/pickr/dist/pickr.min.js:10:3913)",
    "at x._parseLocalColor (/node_modules/@simonwep/pickr/dist/pickr.min.js:12:11236)",
    "at x.setColor (/node_modules/@simonwep/pickr/dist/pickr.min.js:12:13537)",
    "at h.View.eval (/imports/ui/pages/editproject.js:1:2013)",
    "at https://xxx/2f58496c0c10a3f094805fdeb367f45ca604382d.js?meteor_js_resource=true:136:19108",
    "at Function.t._withTemplateInstanceFunc (https://xxx/2f58496c0c10a3f094805fdeb367f45ca604382d.js?meteor_js_resource=true:136:38048)",
    "at https://xxx/2f58496c0c10a3f094805fdeb367f45ca604382d.js?meteor_js_resource=true:136:19059",
    "at Object.h._withCurrentView (https://xxx/2f58496c0c10a3f094805fdeb367f45ca604382d.js?meteor_js_resource=true:136:23216)",
    "at Template.editproject:anonymous (https://xxx/2f58496c0c10a3f094805fdeb367f45ca604382d.js?meteor_js_resource=true:136:19007)",
    "at t._compute (https://xxx/2f58496c0c10a3f094805fdeb367f45ca604382d.js?meteor_js_resource=true:82:2279)"
  ]
}
```

**Comment by faburem** [comment]
now I realise what you did - you removed the value of the color field and that seems to crash the color picker library we are using behind the scenes. We can obviously fix that for titra, but I will also submit a PR to the color picker library to get it fixed there. Thanks again and stay tuned for the fix to arrive in titra v 0.35.0, if you want to workaround the issue for now just select a color using the color picker and save the project, then the error will not be displayed any more.
